db/migrations/20230207100244_products_scents.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE products_scents (
```

Comment:
`product_scents` 테이블로 수정했기 때문에 해당 테이블도 수정해주세요!

```suggestion
CREATE TABLE products_product_scents (
```